cmake/rootcling_wrapper.sh.in

```diff
@@ -77,7 +77,7 @@ LOGFILE=${DICTIONARY_FILE}.log
   -rml ${ROOTMAP_LIBRARY_NAME} \
   ${INCLUDE_DIRS//;/ } \
   ${COMPILE_DEFINITIONS//;/ } \
-  ${HEADERS//;/ } >${LOGFILE} 2>&1
+  ${HEADERS//;/ } 2>&1 | tee ${LOGFILE}
```

Comment:
Instead of using tee I would propose to just cat the log file in case rootcling is failing, ie. 

${HEADERS//;/} >${LOGFILE} 2>&1 || cat ${LOGFILE} >&2

Reply:
good, I have adjusted the PR accordingly